src/components/reporter/Form.js

```diff
@@ -3,79 +3,82 @@ import { StaticQuery, graphql } from "gatsby";
-import { Widgets } from "./widgets";
+import LinesOfCode from "../reporter/LinesOfCodeInput.js";
+import * as widgetStyles from "./widgets/Widgets.module.scss";
+import { Widget } from "./widgets";
+import FormField from "./widgets/FormField";
 
 const config = {
   labelAll: "bug",
-  fields: [
-    {
-      name: "handle",
-      label: "Handle",
-      helpText: "Handle you're competing under (individual or team name)",
-      widget: "warden",
-      required: true,
-      options: [],
-    },
-    {
-      name: "email",
-      label: "Email address",
-      helpText: "Used to send a copy of this form for your records",
-      widget: "text",
-      required: true,
-    },
+};
+
+const emailField = {
+  name: "email",
+  label: "Email address",
+  helpText: "Used to send a copy of this form for your records",
+  widget: "text",
+  required: true,
+};
+
+const addressField = {
+  name: "address",
+  label: "Polygon address",
+  helpText:
+    "Address where your prize should go to (and retroactive token reward should C4 be tokenized later). If you use a smart contract wallet, please contact one of our organizers in Discord in addition to adding the address here.",
+  widget: "text",
+  required: true,
+};
+
+const titleField = {
+  name: "title",
+  label: "Title",
+  helpText:
+    "Summarize your findings for the bug or vulnerability. (This will be the issue title.)",
+  widget: "text",
+  required: true,
+};
+
+const riskField = {
+  name: "risk",
+  label: "Risk rating",
+  widget: "select",
+  required: true,
+  options: [
     {
-      name: "address",
-      label: "Polygon address",
-      helpText:
-        "Address where your prize should go to (and retroactive token reward should C4 be tokenized later). If you use a smart contract wallet, please contact one of our organizers in Discord in addition to adding the address here.",
-      widget: "text",
-      required: true,
+      label: "0 — Gas Optimization",
+      value: "G (Gas Optimization)",
     },
     {
-      name: "title",
-      label: "Title",
-      helpText:
-        "Summarize your findings for the bug or vulnerability. (This will be the issue title.)",
-      widget: "text",
-      required: true,
+      label: "1 — QA (Low and Non-critical)",
+      value: "1 (Low Risk)",
     },
     {
-      name: "risk",
-      label: "Risk rating",
-      widget: "select",
-      required: true,
-      options: [
-        {
-          label: "0 — Gas Optimization",
-          value: "G (Gas Optimization)",
-        },
-        {
-          label: "0 — Non-critical",
-          value: "0 (Non-critical)",
-        },
-        {
-          label: "1 — Low Risk",
-          value: "1 (Low Risk)",
-        },
-        {
-          label: "2 — Medium Risk",
-          value: "2 (Med Risk)",
-        },
-        {
-          label: "3 — High Risk",
-          value: "3 (High Risk)",
-        },
-      ],
+      label: "2 — Medium Risk",
+      value: "2 (Med Risk)",
```

Comment:
We can just drop the numbers from the values I think and just have the label and value both as "Medium Risk"